docs/reference/commandline/daemon.md

```diff
@@ -454,6 +454,14 @@ This example sets the `cgroupdriver` to `systemd`:
 
 Setting this option applies to all containers the daemon launches.
 
+Also Windows Container makes use of `--exec-opt` for special purpose. Docker user
+can specify default container isolation technology with this, for example:
+
+    $ sudo docker daemon --exec-opt isolation=hyperv
```

Comment:
Hm, I don't think this'll work, because we're talking about an option for a *Windows* daemon (I doubt we have `sudo` on Windows).

t.b.h. I don't know what's the way to configure a Windows daemon (do we have documentation for that already?)